examples/apps/colorapp/src/colorteller/Dockerfile

```diff
@@ -3,6 +3,7 @@ WORKDIR /go/src/github.com/aws/aws-app-mesh-examples/colorapp/teller
 
 # Force the go compiler to use modules.
 ENV GO111MODULE=on
+ENV GOPROXY=direct
```

Comment:
Im unsure that we should do this. @bcelenza @kiranmeduri ? This is needed for some corporate networks since golang 1.13 https://golang.org/doc/go1.13#introduction. I think it might be better to just make this configurable instead. We could pull this variable from the host environment when we run Docker build.

plus, you'd really want to make sure this is done for all go examples. I think there are more in this repo.

Reply:
Yeah, I would toggle this based on a [build argument](https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg).

Reply:
Thanks for the pointer. But it doesnt honor the value when setting this as build argument. Keeps failing with certificate issue. Hence not changing. But will add it in all places where go is used.

Reply:
There is one other place - tls-with-acm walkthrough where we use go. There are 2 things here:
1. The script uses 1.10 version of go which doesnt have this problem. 
2. Even after updating to latest version, it doesnt fail with certificate issue. 
Hence modified to consume latest version of go in the other script.

Reply:
@rajal-amzn I dont know what you mean by:

> But it doesnt honor the value when setting this as build argument

Youd have to update the Dockerfile to accept the build argument. Something like this:

```Dockerfile
ARG GOPROXY=https://proxy.golang.org # this is the default
ENV GOPROXY=${GOPROXY}
```

Then the `docker build` command would get an optiona `--build-arg GOPROXY`

Reply:
Thanks. I had missed setting it as ARG and tried passing from builder arg. Thats why it didnt work previously. Fixed now.